config/events/config.go

```diff
@@ -0,0 +1,43 @@
+package events
+
+import (
+	"github.com/upbound/upjet/pkg/config"
+)
+
+// Configure adds configurations for events group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_cloudwatch_event_permission", func(r *config.Resource) {
+		r.References["event_bus_name"] = config.Reference{
+			Type:              "github.com/upbound/provider-aws/apis/events/v1beta1.Bus",
+			RefFieldName:      "EventBusNameRefs",
+			SelectorFieldName: "EventBusNameSelector",
```

Comment:
No need to specify the referencer field names here as we are using the default names generated for them (assuming `EventBusNameRef` instead of `EventBusNameRefs`):
```suggestion
```

Per our naming conventions, we would expect `EventBusNameRefs` to be an array of references.